docs/use-cases/manage-fleet.md

```diff
@@ -12,6 +12,7 @@ images: ["/fleet/locations.png"]
 ---
 
 You can use Viam's cloud-based fleet management tools to monitor and manage access to your fleet of {{< glossary_tooltip term_id="smart-machine" text="smart machines" >}}.
+Use these tools as you create and scale a new fleet of robots, or [integrate Viam](https://www.viam.com/post/integrate-your-ros2-robot-with-viam) to manage and add functionality to your existing fleet.
```

Comment:
This is a good idea but I think remove the link here and add it to the next steps section instead. you can remove the tutorial one to make space